### PR TITLE
Add edge labels

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -33,6 +33,14 @@
   color: black;
 }
 
+.edge-label {
+  height: 100%;
+  width: 100%;
+  display:flex;
```
Comment (songofroland) [inline src/App.css:39]: `display: flex;`

File: src/logic/dijkstra.ts [hunk 2]
```diff
@@ -76,6 +76,7 @@ function getKeyOfMinValue(obj: Costs): number {
   return parseInt(Object.keys(obj).find((key) => obj[parseInt(key)] === minValue)!);
 }
 
-function arrayToObject(arr: Array<any>, defaultValue: any): any{
+function arrayToObject(arr: Array<any>, defaultValue: any): any {
+  // TODO unexpected use of comma operator - line below - col 61
```
Comment (songofroland) [inline src/logic/dijkstra.ts:80]: Yeah thats pain in the ass

File: src/logic/graphDisassembling.ts [hunk 3]
```diff
@@ -27,8 +27,9 @@ export function getEdgesArray(graph: Graph): Array<Edge> {
 
   for (let from = 0; from < graph.length; from++) {
     for (let to = 0; to < graph[from].length; to++) {
-      const edge = new Edge(from, to);
-      if (edge.from !== edge.to && !isDuplicate(edge)) {
+      const cost = graph[from][to];
+      const edge = new Edge(from, to, String(cost));
+      if (edge.from !== edge.to && !isDuplicate(edge) && cost > 0) {
```
Comment (songofroland) [inline src/logic/graphDisassembling.ts:32]: May there be cost <= 0 ?

**Review by songofroland — APPROVED** [review]

File: src/logic/graphDisassembling.ts (hunk 3 above)
Comment (wilmhit) [reply src/logic/graphDisassembling.ts:32]: Yes there can be cost 0. Means edge is non-existant

File: src/logic/dijkstra.ts (hunk 2 above)
Comment (wilmhit) [reply src/logic/dijkstra.ts:80]: Idk, looks good to me. Doesn't work without comma. Gonna do eslint-ignore-next-line or w/e its called

**Pull request merged**